Updates to Subscription section (#25041)
docker/docker.github.io: content/manuals/subscription/scale.md

---
title: Scale your subscription
linkTitle: Scale
description: Scale Docker Build Cloud and Testcontainers Cloud consumption for your subscription
keywords: scale subscription, docker build cloud minutes, testcontainers cloud minutes, usage scaling
weight: 30
---

Docker subscriptions let you scale consumption as your needs grow. All paid Docker subscriptions include base amounts of Docker Build Cloud build minutes and Testcontainers Cloud runtime minutes that you can supplement with additional capacity.

You can scale consumption for:

- Docker Build Cloud build minutes
- Testcontainers Cloud runtime minutes

> - Docker Hardened Images (DHI) repositories with Docker Select

Reviewer: **suggestion:** There is no such thing as `Docker Select`. `Select` is a plan name for Docker Hardened Images (DHI) product.

```suggestion
- Docker Hardened Images (DHI) repositories
```

- Use unlimited runtime minutes on-demand with billing at the end of each monthly cycle

On-demand usage is billed at higher rates than pre-purchased capacity. To avoid higher on-demand charges, pre-purchase additional minutes if you expect consistent usage over your subscription's included minutes.

## Add DHI repositories

> To add more DHI repositories to your Docker Select plan:

Reviewer: **suggestion:** same as above - ~`Docker Select`~

```suggestion
To add more hardened repositories to your DHI Select plan:
```

1. Sign in to [Docker Home](https://app.docker.com/) and select your
   organization.
1. Select **Billing**.
1. On the Overview page, select **Manage** next to **Hardened Images**.

> 1. Select how many mirrored repositories the account can use.

Reviewer: **suggestion:** Each DHI repo a user purchases allows the org owner to mirror a selected hardened image from a public catalog to an organization namespace. Therefore, I would phrase it something like this...

```suggestion
1. Select how many repositories the account can mirror.
```

>    - You can select between one and seven mirrors.

Reviewer 2: Based on @ob1dev's comment in the Scale topic, there's no limit on the number of mirrors. Could you also update this section
